### PR TITLE
Deletes a unused spookoween file and the related sound folder. Saves a few features off of it.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
We have a halloween .dm file (the one with spoopy closets with invincible mobs which could easily be exploited and that gets old and annoying quickly) that hasn't been used in years. I'm deleting it and its relative sound folder (only a few sound files were unused, the rest was relocated in other folders). I'm saving the candy corn crossbow and the hack-o-lantern as a hallowen exclusive cosmetic change for the energy crossbow and cryptographic sequencer uplink items, and coffin and skull cookies are now craftable.

## Why It's Good For The Game
Binning an unused file with honestly vexatious gimmicks (I know because I have played on a server with it included years ago) that's no more included in the .dme during halloween time while saving the few neat things it had.

## Changelog

:cl: Ghommie (taken from an old unused file)
expansion: Skull and coffin cookies can now be baked.
expansion: Energy Crossbows and Cryptographic sequencers now have a Halloween exclusive look.
/:cl:

### CODE REVIEW

File: code/modules/holiday/halloween.dm [hunk 1]
```diff
@@ -1,276 +0,0 @@
...
-//spooky recipes
-
-/datum/recipe/sugarcookie/spookyskull
-	reagents_list = list(/datum/reagent/consumable/flour = 5, /datum/reagent/consumable/sugar = 5, /datum/reagent/consumable/milk = 5)
-	items = list(
-		/obj/item/food/egg,
-	)
-	result = /obj/item/food/cookie/sugar/spookyskull
-
-/datum/recipe/sugarcookie/spookycoffin
```
Comment (Fikou) [inline code/modules/holiday/halloween.dm:15]: dont remove these

File: code/modules/holiday/halloween.dm [hunk 2]
```diff
@@ -1,276 +0,0 @@
...
-
-/mob/living/simple_animal/shade/howling_ghost/adjustHealth(amount, updating_health = TRUE, forced = FALSE)
-	. = 0
-
-///////////////////////////
-//Spookoween Insane Clown//
-///////////////////////////
-
-///Insane clown mob. Basically a clown that haunts you.
-/mob/living/simple_animal/hostile/clown_insane
```
Comment (Fikou) [inline code/modules/holiday/halloween.dm:179]: nooo not the floor cluwne

Comment (Ghommie) [reply code/modules/holiday/halloween.dm:179]: Eheh, you wish it's actually a floor cluwne.

**Comment by Mothblocks** [comment]
whaaaat you could have bats fly out of lockers? thats funny we should bring that back as like particles or something some other time, reminds me of apex's halloween crates

**Comment by Ghommie** [comment]
> 
> 
> whaaaat you could have bats fly out of lockers? thats funny we should bring that back as like particles or something some other time, reminds me of apex's halloween crates

If someone finds me a good bats screeching/flying sound effect shorter than 12 seconds, perhaps. Though it'd be better if it was its own PR.

**Comment by Mothblocks** [comment]
yeah its own pr i just didnt know that was a thing

**Pull request merged**